resources/assets/js/components/JobBuilder/Review/JobReview.tsx

```diff
@@ -413,6 +416,7 @@ export const JobReviewDisplay: React.FC<JobReviewDisplayProps> = ({
           description="Description under primary title of review section"
         />
       </p>
+      <JobReviewActivityFeed jobId={job.id} isHrAdvisor={isHrAdvisor} />
```

Comment:
I think this should be extracted out of JobReviewDisplay - that component should only be responsible for the job summary information. <JobReviewActivityFeed> should be inserted just above <JobReviewDisplay> in <JobReview>, and in JobReviewHrPage.